### ISSUE TITLE
test against xarray HEAD

### ISSUE THREAD

**Issue by dcherian** [dcs4cop/xcube#460]
Testing against Xarray HEAD would've caught https://github.com/pydata/xarray/issues/5329 a month or more ago. Would it be possible to set that up? It has usually helped catch a lot of pre-release bugs.

**Comment by forman** [comment]
@dcherian I agree that this would help finding issues with xarray early, but currently this will produce to much overhead for us. We may think about about a dedicated branch that we'll keep up-to-date with our master with the difference that we use xarray HEAD dependency.
